backend/setupWebSession.ts

```diff
@@ -13,13 +14,23 @@ export default () => {
     const { enabled, host, port, password } = config.redis
     if (!enabled || !host) return null
 
-    const client = redis.createClient({
+    const client = createClient({
       host,
       port,
       password,
       tls: config.app.production ? {} : false,
+      socket: {
+        reconnectStrategy: (attempts: number) => {
```

Comment:
I think this is a new configuration option for v4 so this is unlikely to have an effect (though you will need it in v4).

It might be that the v3 version is at the top level rather than within `socket` but I think v3 has a default retry strategy which is reasonable anyway - `Math.min(retries * 50, 500)`.

Reply:
(In v4 the intent was to have the same default, there's just a [bug](https://github.com/redis/node-redis/issues/2095) that meant its not working - if I'd known that that was the default I probably would have gone with that rather than being fancy 🤷‍♂️ )